Enables Novel Posting

R3dtail opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Doubles the emote limit from 2048 to 4096.

## Why It's Good For The Game

  Emoting is good, increasing the ability of people to write to their hearts content is also good.  

## Changelog
:cl:

tweak: tweaked a few things
code: changed some code
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

R3dtail committed:
Enables Novel Posting

timothyteakettle commented on code/__DEFINES/say.dm:
```diff
@@ -86,7 +86,7 @@
 //Don't set this very much higher then 1024 unless you like inviting people in to dos your server with message spam
-#define MAX_MESSAGE_LEN			2048		//Citadel edit: What's the WORST that could happen?
+#define MAX_MESSAGE_LEN			4096		//Citadel edit: What's the WORST that could happen?
 #define MAX_FLAVOR_LEN			4096		//double the maximum message length.
```
just remove the comment here as it's no longer true

R3dtail committed:
Update say.dm

Deleted the comment on line 90 since max_message_len and max_flavor_len  are now equal as it was a requested change
